Services/Simulations.cs

```diff
@@ -367,6 +378,75 @@ public async Task DeleteAsync(string id)
             await this.simulationsStorage.DeleteAsync(id);
         }
 
+        public async Task TrySeedAsync(string templateName)
+        {
+            try
+            {
+                // Skips if the seeding has already been complete
+                if (!await this.mainStorage.ExistsAsync(SEED_STATUS_KEY))
+                {
+                    await this.SeedSimulationsAsync(templateName);
+                    var record = new StorageRecord { Id = SEED_STATUS_KEY, Data = "Seed Completed" };
+                    await this.mainStorage.CreateAsync(record);
+                }
+                else
+                {
+                    this.log.Info("Seeding skipped.");
+                }
+            }
+            catch (Exception e)
+            {
+                var msg = "Failed to seed default simulations." + e.Message;
+                this.log.Error(msg);
+                this.diagnosticsLogger.LogServiceError(msg);
+            }
+        }
+
+        // This creates sample simulations that will be shown on simulation dashboard by default
+        private async Task SeedSimulationsAsync(string templateName)
+        {
+            string content;
+            var fileName = templateName + ".json";
+            var root = Path.GetDirectoryName(Assembly.GetEntryAssembly().Location);
+            var filePath = Path.Combine(root, "data", "templates", fileName);
```

Comment:
see if constants can make it easier to understand where templates are stored. See also how device models are loaded